pipeline-steps/bootstrap.yaml

```diff
@@ -19,9 +19,12 @@ steps:
       addSpnToEnvironment: true
       scriptLocation: inlineScript
       failOnStandardError: 'true'
-      inlineScript: |   
-        echo "##vso[task.setvariable variable=AZURE_MI_ID]$(az identity show --resource-group genesis-rg --name aks-$(env)-mi --query="clientId" -o tsv)"
-        echo "##vso[task.setvariable variable=AKS_ISSUER_URL]$(az aks show -n cft-${{ parameters.environment }}-${{ parameters.cluster }}-aks -g cft-${{ parameters.environment }}-${{ parameters.cluster }}-rg --query "oidcIssuerProfile.issuerUrl" -otsv)"
+      inlineScript: |
+        echo "##vso[task.setvariable variable=AZURE_MI_ID]$(az identity show --resource-group genesis-rg --name aks-${{ parameters.environment }}-mi --query="clientId" -o tsv)"
+
+        if [[ "$(clusters)" != "All" ]]; then
+          echo "##vso[task.setvariable variable=AKS_ISSUER_URL]$(az aks show -n cft-${{ parameters.environment }}-${cluster_numbers}-aks -g cft-${{ parameters.environment }}-${cluster_numbers}-rg --query "oidcIssuerProfile.issuerUrl" -otsv)"
+        fi
```

Comment:
Is clusters right here?
Seems it should be using `parameters.cluster` instead?

Similar story with cluster_numbers var